DaS-PC-MPChan/DSCM.Designer.vb

```diff
@@ -188,7 +188,6 @@ Partial Class DSCM
         Me.nmbMaxNodes.Font = New System.Drawing.Font("Microsoft Sans Serif", 9.75!, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, CType(0, Byte))
         Me.nmbMaxNodes.Location = New System.Drawing.Point(737, 3)
         Me.nmbMaxNodes.Maximum = New Decimal(New Integer() {32, 0, 0, 0})
-        Me.nmbMaxNodes.Minimum = New Decimal(New Integer() {10, 0, 0, 0})
```

Comment:
DSCM isn't a tool to help play offline.  That feature already exists just by playing the game "offline".

Also, running online with 0 max nodes would make online functionality worse for others.  The servers would match you up with players and then you'd immediately disconnect them.